ansible/modules/hashivault/hashivault_delete.py

```diff
@@ -78,9 +83,13 @@ def main():
 def hashivault_delete(params):
     result = { "changed": False, "rc" : 0}
     client = hashivault_auth_client(params)
+    version = params.get('version')
     secret = params.get('secret')
     if secret.startswith('/'):
         secret = secret.lstrip('/')
+
```

Comment:
I'm going to have to shuffle this around because this looks like it won't support full path name type secret names.  I should make sure there is a test.